akka-cluster/src/test/scala/akka/cluster/ClusterTestKit.scala

```diff
@@ -0,0 +1,136 @@
+package akka.cluster
+
+import akka.actor.{ ActorSystem, Address }
+import akka.cluster.MemberStatus.Removed
+import akka.testkit.TestKitBase
+import com.typesafe.config.{ Config, ConfigFactory }
+
+import scala.collection.{ immutable ⇒ im }
+
+/**
+ * Builds on TestKitBase to provide some extra utilities to run cluster test.
+ *
+ * All functionality is provided through ClusterTest stateful class.
+ */
+trait ClusterTestKit extends TestKitBase {
```

Comment:
Added this utility class to help me manage the actor system and cluster formation in single jvm test. 

This could be useful for other tests we have.